doc/rados/configuration/ceph-conf.rst

```diff
@@ -572,14 +575,14 @@ example, to enable the debug log level on a specific OSD, run a command of this
 .. note:: If an option has been customized in a local configuration file, the
    `central config
    <https://ceph.io/en/news/blog/2018/new-mimic-centralized-configuration-management/>`_
-   setting will be ignored (it has a lower priority than the local
+   setting will be ignored (because it has a lower priority than the local
    configuration file).
 
 Override values
 ---------------
 
-Options can be set temporarily by using the `tell` or `daemon` interfaces on
-the Ceph CLI. These *override* values are ephemeral, which means that they
+Options can be set temporarily by using the ``tell`` or ``daemon`` interfaces
+on the Ceph CLI. These *override* values are ephemeral, which means that they
```

Comment:
by using the Ceph CLI ``tell`` or ``daemon`` commands.

Reply:
Accepted.